numpy/__init__.py

```diff
@@ -228,6 +241,13 @@
         # module level getattr is only supported in 3.7 onwards
         # https://www.python.org/dev/peps/pep-0562/
         def __getattr__(attr):
+            # Raise AttributeError for expired attributes
+            try:
+                msg = __expired_attrs__[attr]
+                raise AttributeError(msg)
+            except KeyError:
+                pass
```

Comment:
nit for consistency with the code below:
```suggestion
            except KeyError:
                pass
            else:
                raise AttributeError(msg)
```

Reply:
Fixed in an update.